File: 3. Implementation/main.cpp
#include <iostream>
#include "Calculator.h"

using namespace std;

int main(){

    int r;
    Calculator c01;

    r = c01.add();
    cout << r;

    r = c01.substract();
    cout << r;

    r = c01.multiply();
    cout << r;

    r = c01.divide();
    cout << r;

    return 0;
}

File: 3. Implementation/main.py
# a simple example
import Calc

input01 = int(input("Enter the 1st integer: "))
input02 = int(input("Enter the 2nd integer: "))
calc01 = Calc.Calculator(input01, input02)

calc01.plus()
calc01.minus()
calc01.multiply()
calc01.divide()
